mise.toml

```diff
@@ -0,0 +1,62 @@
+#:schema https://mise.jdx.dev/schema/mise.json
+
+[tools]
+uv = "0.5.1"
+
+[tasks."setup"]
+description = "Set up the dev environment (install Rust, etc)"
+run = [
+  "rustup toolchain install 1.84",
+  # We want rustfmt from nightly
+  "rustup toolchain install nightly --profile minimal --component rustfmt",
+  # Tools installed via Cargo
+  "cargo +1.84 install --locked bacon",
+  "cargo +1.84 install --locked cargo-insta",
+  "cargo +1.84 install --locked cargo-nextest",
+]
+
+[tasks."format"]
+description = "Format the code"
+run = "cargo +nightly fmt"
+
+[tasks."lint"]
+description = "Lint the code"
+run = "cargo +1.84 clippy --workspace --all-targets"
+
+[tasks."lint:ci"]
+description = "Lint the code using the same clippy version as CI would"
+run = [
+  # Check for cmake
+  "cmake --version || echo 'cmake is not installed, cargo clippy might fail'",
+  # `cargo +stable` doesn't check for the latest stable, so we have to discover it.
+  # We just extract versions from the output of `rustup check` and take the latest.
+  '''
+  STABLE=$(rustup check | grep '^stable' | grep -Eo '\d+\.\d+\.\d+' | sort --version-sort | tail -1)
+  cargo +$STABLE clippy --all-features --workspace --all-targets -- -D warnings
+  ''',
+]
+
+[tasks."zizmor"]
```

Comment:
suggestion: `zizmor` is the specific tool name, while the rest of the task names are pretty abstract. I'd rename this to perhaps `check:workflows` (as per above), or rename other stuff to e.g. `clippy`.